.github/workflows/ci.yaml

```diff
@@ -32,13 +32,19 @@ env:
 
 jobs:
   controller-test:
+    env:
+      REPOSITORY: ${{ github.event.pull_request.head.repo.full_name }}
+      REF: ${{ github.event.pull_request.head.ref }}
```

Comment:
Does this need to duplicated for the `scorecard-test` job? Or can it be moved to the root `env` property instead?

Reply:
Opps! I thought I put it there. Will fix it!

Reply:
Updated thanks!